scripts/helm/test-publish-chart-yaml.sh

```diff
@@ -157,15 +170,17 @@ test_one "Develop is special"
 
 CHECK_BRANCH=main
 DATE_TIME=$(date +"$DATE_TIME_FMT")
-APP_TAG=0.0.0-$DATE_TIME
-CHART_VERSION=0.0.0
-CHART_APP_VERSION=0.0.0
+github_latest_plus_one=$(github_latest_version_plus_one)
+APP_TAG=$github_latest_plus_one-$DATE_TIME
+CHART_VERSION=$github_latest_plus_one
+CHART_APP_VERSION=$github_latest_plus_one
 test_one "Main is special"
 
 CHECK_BRANCH=main
-APP_TAG=0.0.0-main
-CHART_VERSION=0.0.0
-CHART_APP_VERSION=0.0.0
+github_latest_plus_one=$(github_latest_version_plus_one)
```

Comment:
we should hardcode things to test here, rather copying and pasting the same code UT other it makes the test itself redundant..

Reply:
I agree, alright.

Reply:
Because the value is changing all of the time.

Reply:
Why can't it be hardcoded?

Reply:
Alright, I can add an input to the test. Okay.

Reply:
Added a test.
Testing with explicit input.
One condition cannot be tested, as it depends on the branches.